README.md

```diff
@@ -67,24 +46,9 @@ channels:
 - [Attend our meetings](https://tockify.com/cncf.public.events/monthly?search=Contributor+Strategy)
   - We meet once a month, and we'd love to have you join us!
```

Comment:
```suggestion
  - We have a general TAG meeting once a month and many of our Working Groups / Teams also have regular meetings. We'd love to have you join us!
```

I know this is out of scope for the charter updates, but it seems like we should be clear that we have more than one meeting per month.